README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portfolio
 
-> This project is my portfolio website. The website is live at 
+> This project is my portfolio website. It is live at 
```

Comment:
## [OPTIONAL]

- [ ] Kindly your README  would look more professional if you include the section of the ***live demo*** as the microverse README template is structured and removing it from the description section. Just like this :point_down: 
![Screenshot from 2021-12-06 12-43-06](https://user-images.githubusercontent.com/65068771/144832871-62402c3f-108d-4d10-aade-d34d4034b152.png)

- [ ] I would also recommend putting only the screenshot of the first section of your portfolio instead of the full screenshot of the project. This will prevent visitors of your README from infinite scrolling. :muscle: I know you can do it :clap: :clap: